SlicerCART/src/SlicerCART.py

```diff
@@ -1323,7 +1368,19 @@ def push_save(self):
             with open(LABEL_CONFIG_FILE_PATH, 'w') as file:   
                 yaml.safe_dump(self.label_config_yaml, file)
 
-            self.close()
+       self.segmentation_config_yaml['is_display_timer_requested'] = self.display_timer_checkbox.isChecked()
+
+       with open(SEGMENTATION_CONFIG_FILE_PATH, 'w') as file:   
+            yaml.safe_dump(self.segmentation_config_yaml, file)
+
+       if self.edit_conf and self.segmenter.outputFolder is not None and os.path.exists(f'{self.segmenter.outputFolder}{os.sep}{CONF_FOLDER_NAME}'): 
```

Comment:
Is it possible to explain the code below?

Reply:
Although I didn't originally write this part, I believe that it is used to fetch configuration files if in the middle of segmentation or use case, the user decides to click on `Edit SlicerCART Configuration`

Reply:
Ok!